wiki/People/osu!monthly_Team/en.md

```diff
@@ -0,0 +1,40 @@
+# osu!monthly Team
+
+The **osu!monthly Team** manages the [osu!monthly](/wiki/Community/osu!monthly) newsletter.
+
+The contributors of [osu!monthly](/wiki/Community/osu!monthly) are active players in the community, keeping track of what is going on each month and curating the content to be included in each issue.
+
+## Team members
+
+### Main contributors
+
+The people listed below are the core of the osu!monthly Team and participate every month in the production of the news post.
+
+| Name | Role |
+| :-- | :-- |
+| ::{ flag=RS }:: [0x84f](https://osu.ppy.sh/users/7944724) | Maintainer |
+| ::{ flag=US }:: [Blushing](https://osu.ppy.sh/users/5927823) | Co-maintainer |
+| ::{ flag=GB }:: [chromb](https://osu.ppy.sh/users/10238680) | Trivia Tinkerer |
+| ::{ flag=CA }:: [ianpep](https://osu.ppy.sh/users/7699030) | Writer[^task-taiko] |
+| ::{ flag=ES }:: [kadoen](https://osu.ppy.sh/users/12780575) | Writer[^task-mania] |
+| ::{ flag=ES }:: [Patatitta](https://osu.ppy.sh/users/16294034) | Writer[^task-catch] |
+| ::{ flag=FR }:: [Pisapou](https://osu.ppy.sh/users/16640021) | Co-maintainer |
+| ::{ flag=ES }:: [RandomeLoL](https://osu.ppy.sh/users/7080063) | Writer[^task-mania] |
+| ::{ flag=GB }:: [Tanza3D](https://osu.ppy.sh/users/10379965) | Graphic designer[^task-Tanza3D] |
+| ::{ flag=ID }:: [Wowcake](https://osu.ppy.sh/users/16121851) | Helper |
+
+### Other contributors
+
+The people listed below remain vital to the osu!monthly even if they do not write for it, as they provide information in addition to the main contributors.
+
+| Name | Affiliated section |
+| :-- | :-- |
+| ::{ flag=DE }:: [RockRoller](https://osu.ppy.sh/users/8388854) and the [skinship team](https://skinship.xyz/) | Skinning recap |
+| ::{ flag=FR }:: [Xahlt](https://osu.ppy.sh/users/13466351) | Mapping recap |
+
+## Notes
+
+[^task-taiko]: Mainly handles writing osu!taiko-related sections
+[^task-mania]: Mainly handles writing osu!mania-related sections
+[^task-catch]: Mainly handles writing osu!catch-related sections
+[^task-Tanza3D]: Creates the osu!monthly banners
```

Comment:
maybe use adjectives ("mainly responsible for ...") or nouns here instead of verbs, or it reads incomplete
```suggestion
[^task-taiko]: Area of activity: osu!taiko-related sections
[^task-mania]: Area of activity: osu!mania-related sections
[^task-catch]: Area of activity: osu!catch-related sections
[^task-Tanza3D]: Area of activity: the osu!monthly banners
```

Reply:
sounds a bit too terse for a note imo, i think the alternative would sound better:

```suggestion
[^task-taiko]: Responsible for osu!taiko sections
[^task-mania]: Responsible for osu!mania sections
[^task-catch]: Responsible for osu!catch sections
[^task-Tanza3D]: Creates the osu!monthly banners
```

also not sure if the last note is that necessary

i mean you could also do something like this:

```markdown
| ::{ flag=CA }:: [ianpep](https://osu.ppy.sh/users/7699030) | Writer (osu!taiko) |
| ::{ flag=ES }:: [kadoen](https://osu.ppy.sh/users/12780575) | Writer (osu!mania) |
| ::{ flag=ES }:: [Patatitta](https://osu.ppy.sh/users/16294034) | Writer (osu!catch) |
| ::{ flag=FR }:: [Pisapou](https://osu.ppy.sh/users/16640021) | Co-maintainer |
| ::{ flag=ES }:: [RandomeLoL](https://osu.ppy.sh/users/7080063) | Writer (osu!mania) |
| ::{ flag=GB }:: [Tanza3D](https://osu.ppy.sh/users/10379965) | Graphic designer |
```